Add information for lib3to6 to practicalities (#287)
python3statement/python3statement.github.io: _practicalities/intro.md

support, it does perfectly apply to a package that wishes to not support Python 3,
or is stopping support for any minor version. 

This page gathers information and links to resources allowing a library
to stop supporting an older version of Python without causing too

> much disruption for users who haven't upgraded to this new version.

Reviewer: # _~~~~_
```

check](https://github.com/ipython/ipython/blob/6a3e2db0c299dc05e636653c4a43d0aa756fb1c8/setup.py#L23-L58)
that attempt to detect which version of pip is in used.

## If your users absolutely need Py2.7 but you want to move on

If you have users that still depend on Python 2.7 and you cannot drop support yet, 

> it may nonetheless be possible to start using Python 3 in your codebase. T

Reviewer 2: ```suggestion
it may nonetheless be possible to start using Python 3 in your codebase.
```

> hrough automatic conversion using [lib3to6](https://pypi.org/project/lib3to6/) 

Reviewer 2: ```suggestion
Through automatic conversion using [lib3to6](https://pypi.org/project/lib3to6/) 
```

> (similar to BabelJS for JavaScript) 

Reviewer 2: Looks like it's named simply Babel: https://babeljs.io/ 

```suggestion
(similar to Babel for JavaScript) 
```

> you can generate distribution files that will work on both Python2 and Python3.

Reviewer 2: ```suggestion
you can generate distribution files that will work on both Python 2 and Python 3.
```

> Lib3to6 is also useful if your project needs to support older versions of Python3

Reviewer 2: ```suggestion
Lib3to6 is also useful if your project needs to support older versions of Python 3
```

explain why they can't work and what are their drawbacks before you attempt to
implement them.

### Don't use a meta-package

It is possible to release a meta-package that has _virtually_ no code and relies
on a conditional dependency to install its actual core code on the user system.
For example, Frob-6.0 could be a meta-package which depends on
Frob-real-py2 on Python < 3.0, and Frob-real-py3 on Python ≥ 3.4. While
this approach is _doable_ this can make imports confusing.

> ## Don't depend on setuptools

Reviewer 3: I think this one is 

```suggestion
## Do depend on setuptools>=24.3
```

Maybe ?

Author: How about `Don't depend on setuptools unless version >=24.3`

So that it's all a list of not to dos?

Reviewer 3: sounds good updated suggestion.

Reviewer 4: buildscript {
    repositories {
        // Gradle 4.1 and higher include support for Google's Maven repo using
        // the google() method. And you need to include this repo to download
        //
